test/OrigamiMembershipToken.t.sol

```diff
@@ -596,6 +596,56 @@ contract MembershipTokenVotingPowerTest is OMTHelper {
         assertEq(token.delegates(mintee), delegatee);
     }
 
+    function testTransferMTTransfersVotingPower() public {
+        address other = address(0x7);
+
+        // mint some more tokens as owner
+        token.enableTransfer();
+        token.safeMint(mintee);
+
+        // self-delegate
+        vm.prank(other);
+        token.delegate(other);
+
+        assertEq(token.getVotes(mintee), 1);
+        assertEq(token.getVotes(other), 1);
+
+        // transfer 10 tokens to mintee
+        vm.prank(other);
+        token.transferFrom(other, mintee, 1);
+
+        // check that mintee has 2 votes
+        assertEq(token.getVotes(mintee), 2);
+
+        // check that other has 0 votes
+        assertEq(token.getVotes(other), 0);
+    }
+
+    function testTransferMTWhenDelegated() public {
```

Comment:
```suggestion
    function testTransferVotingUnitsWhenDelegated() public {
```

Reply:
This isn't voting units. This is an actual transfer of the Membership Token.

Reply:
Ok to resolve? @mrmemes-eth

Reply:
Hmm... I still don't like the `MT` in the name, it doesn't tell me what this does. Maybe:

```suggestion
    function testTransfersWhenDelegated() public {
```